modules/ui/init.js

```diff
@@ -665,13 +666,14 @@ export function uiInit(context) {
             .triggerType(triggerType)
             .operations(operations);
 
-        // render the menu
-        context.map().supersurface.call(_editMenu);
+        // render the menu onto the overmap
+        overMap
+            .call(_editMenu);
     };
 
     ui.closeEditMenu = function() {
         // remove any existing menu no matter how it was added
-        context.map().supersurface
+       overMap
```

Comment:
to fix unit tests (where `closeEditMenu` might be classed without a properly initialized `overMap`):

```suggestion
        if (overMap !== undefined) {
            overMap.select('.edit-menu').remove();
        }
```